.github/workflows/continuous-integration.yml

```
@@ -0,0 +1,21 @@
+name: Continuous Integration
+
+on: [push, pull_request]
```

```suggestion
on: [pull_request]
```

This is our current travis setup (although with github actions would this config build on master?)

---

yes that's the idea. could split into two separate configs